src/app/docs/lookup/lookup-docs.component.html

```diff
@@ -140,12 +247,220 @@
             *skyThemeIf="'modern'" imageSource="~/assets/img/guidelines/lookup/lookup-options-3-modern.png">
         </sky-docs-design-guideline-thumbnail>
     </sky-docs-design-guideline>
+        
+    <sky-docs-design-guideline
+      heading="Custom picker"
+    >
+        <p>
+          When users click the search button or Show all/matches button, the modal that opens displays a simple list of options by default (with checkboxes in the multi-select case). Use a custom picker when a different presentation of the options is valuable, like a 
+          <a *skyThemeIf="'default'" href="https://developer.blackbaud.com/skyux-lists/docs/repeater">repeater</a>
+          <a *skyThemeIf="'modern'" href="https://developer.blackbaud.com/skyux-lists/docs/repeater?svcid=skyux-modern">repeater</a> 
+          or 
+          <a *skyThemeIf="'default'" href="https://developer.blackbaud.com/skyux-angular-tree-component/docs/angular-tree">tree view</a>
+          <a *skyThemeIf="'modern'" >tree view</a>. This can be combined with a custom results template.
+        </p>
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'default'" imageSource="~/assets/img/guidelines/lookup/lookup-options-4-default.png">
+        </sky-docs-design-guideline-thumbnail>
+        <sky-docs-design-guideline-thumbnail
+            *skyThemeIf="'modern'" imageSource="~/assets/img/guidelines/lookup/lookup-options-4-modern.png">
+        </sky-docs-design-guideline-thumbnail>
+    </sky-docs-design-guideline>
 
     </sky-docs-demo-page-section>
 
+    <sky-docs-demo-page-section
+      heading="Behavior and states"
+    >
+      
+        <sky-docs-design-guideline
+          heading="Behavior"
+        >
+            <p class="sky-docs-text-block">
+            When the lookup component gains focus, the menu appears with buttons for “Show all #” and “New”.
```

Comment:
Only the 'New' button is optional, for the purposes of the design guidelines. Adding a comment about "if it is enabled" to the end is a good idea, but any guess I make about how to phrase it will probably be clumsy as my first attempt at the rest of the sentence 😄

Reply:
I don't follow how only New is optional for design guidelines. For the new button, we have a `showAddButton` property that is set to `false` by default, which means that this option only appears when consumers deliberately change the default value of the property, so I get why that is optional. But we also have an `enableShowMore` property that is set to `false` by default. It seems like the exact same scenario.

So I'm pretty sure that both should be portrayed as optional. But if I'm missing something here, then making the distinction would just be a matter of splitting this into two sentences or a compound sentence:
```suggestion
            When users place focus on lookup fields, the lookup menu displays the 'Show all #' button. It also displays the 'New' button if that is enabled.
```

Reply:
Sorry. I just saw your other comment about how even though this is technically optional, we want to present it as required in the design guidelines. Does my second suggested rewrite work?

Also, FWIW, I think it's confusing to pretend this option is required in the design guidelines when it's actually optional AND turned off by default. Is it worth adding an item to explain that while it's technically optional, we strongly advise against NOT using it?